### PR TITLE
nginx: add service resource to nginx_site resource

### PR DESCRIPTION
**Pull request by zmaupin**
# Description

This adds the service resource for nginx so `nginx_site` will work.

## Issues Resolved

https://github.com/sous-chefs/nginx/issues/505

## Check List

- [x] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing. # this is not new functionality
- [ ] New functionality has been documented in the README if applicable. # this is not new functionality

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: metadata.rb [hunk 1]
```diff
@@ -6,8 +6,7 @@
 chef_version     '>= 14'
 license          'Apache-2.0'
 description      'Installs and configures nginx'
-long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '10.0.1'
+version          '10.0.2'
```
Comment (Xorima) [inline metadata.rb]: Hey, 
Please don't bump version numbers as we prefer to manage these ourselves 

It is common for us to bundle up releases together etc :)

**Comment by zmaupin** [comment]
Oops sorry! I've already merged. Would you like me to fix that?

**Comment by zmaupin** [comment]
Fixing here https://github.com/sous-chefs/nginx/pull/514

**Pull request merged**